components/resume/company/company.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FC } from 'react'
+import Position from './positions';
+import Skills from './skills';
+
+interface Position {
+    title: string;
+    startDate: string;
+    endDate?: string;
+}
+
+interface CompanyProps {
+    name: string;
+    positions: Position[];
+    startDate: string;
+    endDate?: string;
+    skills: string[];
+    highlights: string[]
+}
+
+const Company: FC<CompanyProps> = ({ name, skills, positions, highlights }) => {
+    return <div>
+        <h3>{name}</h3>
+        <Skills skills={skills} />
```

Comment:
Broke this out as a component responsible for a list of skills since I plan on adding specific styles to space this out and add colors.